linkage_dag.py

```diff
@@ -668,17 +711,23 @@
     (
         last_combination_query
         >> heavy_compute_inputs
-        >> gce_instance_start
+        >> gce_instance_create
+        >> gce_instance_start.as_setup()
         >> prep_environment
         >> create_cset_ids
         >> wait_for_cset_ids
-        >> gce_instance_stop
+        >> gce_instance_stop.as_teardown()
+        >> gce_instance_delete
     )
 
-    gce_instance_start >> run_lid >> gce_instance_stop
+    # Ensure that delete doesn't run if we're in the teardown condition so we'll have a chance to review the outputs
```

Comment:
I don't think I quite understand how the teardown condition blocker works here...but it sounds like you've dealt with it carefully so I believe it does?